net/instaweb/rewriter/critical_css_loader.js

```diff
@@ -39,12 +39,18 @@ pagespeed.CriticalCssLoader.addAllStyles = function() {
 
   var elements = document.getElementsByClassName('psa_add_styles');
 
+    
   for (var i = 0, e; e = elements[i]; ++i) {
-    if (e.nodeName != 'NOSCRIPT') { continue; }
-    var div = document.createElement('div');
-    div.innerHTML = e.textContent;
+    if (e.nodeName == 'NOSCRIPT') { 
+       var div = document.createElement('div');
+       div.innerHTML = e.textContent;
+       var children = div.childNodes;
+       for (var v = 0; v < children.length; ++v) {
+           children[v].removeAttribute('id');
+       }  
     document.body.appendChild(div);
```

Comment:
Nit indentation

Reply:
Indentation spaces changed by tabs

Reply:
Thanks; I proposed one more whitespace change in a4b0e7b, wdyt?

Reply:
You are rigth, I made mistakes when correcting, sorry for that.